docs/projects/blank/overview.md

```diff
@@ -1,3 +1,21 @@
-# Blank Template project
+# Blank Starter Project
 
-// Information about the blank template project
+The Blank starter project is a great starting point for working on the SpatialOS GDK for Unity if you want a clean slate.
+
+Check out the [Blank starter project on Github](https://github.com/spatialos/gdk-for-unity-blank-project)!
+
+## What does it come with?
+
+It has the minimal feature set required to start developing:
```

Comment:
"minimum" better than "minimal" - "minimum" means the absolute smallest amount so works with the rest of the sentence. Minimal means a small amount and doesn't work with the rest of the sentence.

Reply:
added full changes from above